src/cockpit/channels/dbus.py

```diff
@@ -254,7 +254,7 @@ async def get_ready():
                         self.ready(unique_name=self.owner)
                     else:
                         self.close(problem="not-found")
-            task = asyncio.create_task(get_ready())
+            task = self.create_task(get_ready())
             self.tasks.add(task)
```

Comment:
Do we still need `self.tasks` at all?

Reply:
Indeed, we can drop this for a very nice cleanup!  Thanks!